projects/js-packages/components/tsconfig.json

```
@@ -1,5 +1,11 @@
 {
-	"extends": "jetpack-js-tools/tsconfig.base.json",
-	// List all sources and source-containing subdirs.
-	"include": [ "./index.ts", "./components", "./lib", "./tools" ]
+	"extends": "jetpack-js-tools/tsconfig.tsc.json",
+	"include": [ "./index.ts" ],
```

While it looks like it's already built, since it's an entry point in package.json perhaps we should add it here anyway
```suggestion
	"include": [ "./index.ts", "./tools/jp-redirect/index.ts" ],
```